csquotes.sty

```diff
@@ -825,28 +825,39 @@
      \endgroup}
     {#1}}
 
-% \lowercase: workaround for page headers of book.cls et al.
+% The following checks if a string is completely uppercase. If so, it is lowecased,
+% if not, it is returned as is. This works around issues with uppercased language
+% names e.g. in page headers of book.cls et al., while letting mixed-case language
+% names such as USenglish untouched.
+\def\csq@lc@if@uc#1{%
+ \def\tempa{#1}%
```

Comment:
Should be a private temp var, so `\csq@tempa` (and the same for `\tempb`)

Reply:
Done!